# inputs
string fileName
---
# outputs
bool success